experimental/packages/propagator-aws-xray-lambda/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@opentelemetry/propagator-aws-xray-lambda",
+  "version": "1.22.0",
+  "description": "OpenTelemetry AWS Xray propagator provides context propagation for systems that are using AWS X-Ray format.",
+  "main": "build/src/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/propagator-aws-xray-lambda --include-dependencies",
+    "prewatch": "npm run precompile",
+    "compile": "tsc --build tsconfig.json tsconfig.esm.json",
+    "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json",
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "watch": "tsc --build --watch tsconfig.json tsconfig.esm.json",
+    "prepublishOnly": "npm run compile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "awsxray"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "files": [
+    "build/esm/**/*.js",
+    "build/esm/**/*.js.map",
+    "build/esm/**/*.d.ts",
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": ">=1.3.0 <1.9.0"
+  },
+  "devDependencies": {
+    "@babel/core": "7.23.6",
+    "@babel/preset-env": "7.22.20",
+    "@opentelemetry/api": "1.8.0",
+    "@types/mocha": "10.0.6",
+    "@types/node": "18.6.5",
+    "@types/sinon": "10.0.20",
+    "babel-loader": "8.3.0",
+    "babel-plugin-istanbul": "6.1.1",
+    "codecov": "3.8.3",
+    "cross-var": "1.1.0",
+    "lerna": "6.6.2",
+    "mocha": "10.2.0",
+    "nyc": "15.1.0",
+    "sinon": "15.1.2",
+    "ts-loader": "8.4.0",
+    "ts-mocha": "10.0.0",
+    "typescript": "4.4.4"
+  },
+  "dependencies": {
+    "@opentelemetry/propagator-aws-xray": "1.3.1"
+  },
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js/tree/main/propagators/opentelemetry-propagator-aws-xray#readme"
```

Comment:
nit: link is not aligned with the actual package location.

Reply:
The link has been updated.